Add Xavier Pigeon as a new bagger with 3 sessions.

xapn opened the pull request.

xapn committed:
Add Xavier Pigeon as a new bagger with 3 sessions.

dadoonet commented on baggers.js:
```diff
@@ -1,5 +1,90 @@
+          url: 'https://xavierpigeon.com/',
+        },
+        {
+          name: 'Chrysocode',
```
Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

xapn replied:
Done.

dadoonet commented on baggers.js:
```diff
@@ -1,5 +1,90 @@
+          url: 'https://chrysocode.io/',
+        },
+        {
+          name: 'GOST (Gears Of Software Testing)',
```
Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

xapn replied:
Done.

dadoonet commented on baggers.js:
```diff
@@ -1,5 +1,90 @@
+          url: 'https://gearsoftesting.org/',
+        },
+        {
+          name: 'TestAsYouThink Project',
```
Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

xapn replied:
Done.

dadoonet commented on baggers.js:
```diff
@@ -1,5 +1,90 @@
 var data = {
   speakers: [
+    {
```
Please move yourself at the end of the list.

xapn replied:
Done.

dadoonet reviewed (changes requested):
I left some comments. Could you please update your PR?

xapn committed:
Replace URL names and move myself at the end of the baggers list.

- Cf. [guide](https://github.com/brownbaglunch/bblfr_data/#websites).
  The 'Blog' URL name is used in the baggers list but the guide is outdated.
